blocks/layout-grid/src/grid.scss

```diff
@@ -161,7 +161,7 @@ $light-opacity-light-800: rgba(#fff, 0.45);
 	}
 }
 
-/* When clicked, hide the fake drag handles and show the real one */
+// When clicked, hide the fake drag handles and show the real one.
```

Comment:
Yeah I know. But when they're `//` they are stripped from the rendered CSS, whereas `/* */` is kept.